meer-shah㉿SAMEER-SHAH)-[~/Desktop/o-sameer-backup]
└─$ sudo python3 o-sameer.py




































:'#######::::::'######:::::'###::::'##::::'##:'########:'########:'########::
'##.... ##::::'##... ##:::'## ##::: ###::'###: ##.....:: ##.....:: ##.... ##:
##:::: ##:::: ##:::..:::'##:. ##:: ####'####: ##::::::: ##::::::: ##:::: ##:
##:::: ##::::. ######::'##:::. ##: ## ### ##: ######::: ######::: ########::
##:::: ##:::::..... ##: #########: ##. #: ##: ##...:::: ##...:::: ##.. ##:::
##:::: ##::::'##::: ##: ##.... ##: ##:.:: ##: ##::::::: ##::::::: ##::. ##::
. #######:::::. ######:: ##:::: ##: ##:::: ##: ########: ########: ##:::. ##:
:.......:::::::......:::..:::::..::..:::::..::........::........::..:::::..::

Enabling monitor mode on wlan0...
Scanning Wi-Fi networks on: wlan0 for 20 seconds...

Restoring wlan0 to managed mode...
Traceback (most recent call last):
  File "/home/sameer-shah/Desktop/o-sameer-backup/o-sameer.py", line 182, in <module>
    asyncio.run(main())
    ~~~~~~~~~~~^^^^^^^^
  File "/usr/lib/python3.13/asyncio/runners.py", line 195, in run
    return runner.run(main)
           ~~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/asyncio/base_events.py", line 725, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/home/sameer-shah/Desktop/o-sameer-backup/o-sameer.py", line 158, in main
    networks = await async_scan_networks(selected_interface, scan_time=20)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sameer-shah/Desktop/o-sameer-backup/o-sameer.py", line 100, in async_scan_networks
    sniff(iface=interface, prn=packet_handler, timeout=scan_time)
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/scapy/sendrecv.py", line 1424, in sniff
    sniffer._run(*args, **kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/scapy/sendrecv.py", line 1273, in _run
    sniff_sockets[_RL2(iface)(type=ETH_P_ALL, iface=iface,
                  ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              **karg)] = iface
                              ^^^^^^^
  File "/usr/lib/python3/dist-packages/scapy/arch/linux/__init__.py", line 235, in __init__
    _flush_fd(self.ins.fileno())
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/scapy/arch/linux/__init__.py", line 198, in _flush_fd
    os.read(fd, MTU)
    ~~~~~~~^^^^^^^^^
OSError: [Errno 100] Network is down

Restoring wlan0 to managed mode...
